docs/manual/document-format-reference/job-yaml-v12.md

```diff
@@ -833,6 +845,9 @@ Defines a notification for the job. You can include any of `onsuccess`, `onfailu
 
 `onsuccess`/`onfailure`/`onstart`/`onavgduration`/`onretryablefailure`
 
+::: Note: `onavgduration` also requires the following attribute set at the same level as `notification`
+`notifyAvgDurationThreshold` - Add or set a threshold value to the avg duration in order to trigger this notification. Options: - percentage => eg: 20% - time delta => eg: +20s, +20 - absolute time => 30s, 5m Time in seconds if you don't specify time units Can include option value references like ${option.avgDurationThreshold}.
```

Comment:
It looks like this should be above the note? I believe the following section describes the keys under `onsuccess`/`onfailure`/`onstart`/`onavgduration`/`onretryablefailure`, but `notifyAvgDurationThreshold` is its own top-level key.

Reply:
yes, but it's only added in one case of the 5 choices above.  I waffled between both options, but the note text is meant to guide the user to add `notifyAvgDurationThreshold`

Reply:
Do we need changes here or can we see how it works as written?

Reply:
LGTM. The tip could be changed to `warning` if we want to raise the visibility of it.